lib/common.js

```diff
@@ -1,35 +1,68 @@
 var MysqlTypes = {
-  'DECIMAL': 0,
-  'TINY': 1,
-  'SHORT': 2,
-  'LONG': 3,
-  'FLOAT': 4,
-  'DOUBLE': 5,
-  'NULL': 6,
-  'TIMESTAMP': 7,
-  'LONGLONG': 8,
-  'INT24': 9,
-  'DATE': 10,
-  'TIME': 11,
-  'DATETIME': 12,
-  'YEAR': 13,
-  'NEWDATE': 14,
-  'VARCHAR': 15,
-  'BIT': 16,
+  DECIMAL: 0,
+  TINY: 1,
+  SHORT: 2,
+  LONG: 3,
+  FLOAT: 4,
+  DOUBLE: 5,
+  NULL: 6,
+  TIMESTAMP: 7,
+  LONGLONG: 8,
+  INT24: 9,
+  DATE: 10,
+  TIME: 11,
+  DATETIME: 12,
+  YEAR: 13,
+  NEWDATE: 14,
+  VARCHAR: 15,
+  BIT: 16,
   // Fractional temporal types in MySQL >=5.6.4
-  'TIMESTAMP2': 17,
-  'DATETIME2': 18,
-  'TIME2': 19,
-  'NEWDECIMAL': 246,
-  'ENUM': 247,
-  'SET': 248,
-  'TINY_BLOB': 249,
-  'MEDIUM_BLOB': 250,
-  'LONG_BLOB': 251,
-  'BLOB': 252,
-  'VAR_STRING': 253,
-  'STRING': 254,
-  'GEOMETRY': 255,
+  TIMESTAMP2: 17,
+  DATETIME2: 18,
+  TIME2: 19,
+  NEWDECIMAL: 246,
+  ENUM: 247,
+  SET: 248,
+  TINY_BLOB: 249,
+  MEDIUM_BLOB: 250,
+  LONG_BLOB: 251,
+  BLOB: 252,
+  VAR_STRING: 253,
+  STRING: 254,
+  GEOMETRY: 255
+};
+
+var MysqlCodeTypes = {
```

Comment:
V8 can do a better job of optimizing a fixed length object literal than it will doing it dynamically, but that's not a good argument to keep it this way.

The only solid reason why I generally do this is so that code introspection/autocomplete in IDEs will be able to find the values.

I have used those types of functions int he past for larger lists that were remotely loaded. I'd say we should just move all these definitions into a file where there's no code.